de.peeeq.wurstscript/src/main/java/de/peeeq/wurstio/languageserver/requests/CodeActionRequest.java

```diff
@@ -61,32 +62,38 @@ public CodeActionRequest(CodeActionParams params, BufferManager bufferManager) {
             return Collections.emptyList();
         }
         // get element under cursor
-        Element e = Utils.getAstElementAtPos(cu, line, column, false);
+        Optional<Element> e = Utils.getAstElementAtPos(cu, line, column, false);
 
 
         WLogger.info("Code action on element " + Utils.printElementWithSource(e));
-        if (e instanceof ExprNewObject) {
-            ExprNewObject enew = (ExprNewObject) e;
+        if (e.isEmpty()) {
+            // TODO non simple TypeRef
```

Comment:
Where did this line come from?

Reply:
Ah it's copy-pasted from the final return in this function. Unclear to me if the TODO is relevant here as well.